pkg/trace/stats/statsraw.go

```diff
@@ -172,24 +189,10 @@ func (sb *RawBucket) add(s *WeightedSpan, aggr Aggregation) {
 		gs.errors += s.Weight
 	}
 	gs.duration += float64(s.Duration) * s.Weight
-	// alter resolution of duration distro
-	trundur := nsTimestampToFloat(s.Duration)
+	secondsDuration := float64(s.Duration)/1e9
 	if s.Error != 0 {
-		gs.errDistribution.Add(trundur)
+		gs.errDistribution.Add(secondsDuration)
 	} else {
-		gs.okDistribution.Add(trundur)
-	}
-}
-
-// 10 bits precision (any value will be +/- 1/1024)
-const roundMask int64 = 1 << 10
-
-// nsTimestampToFloat converts a nanosec timestamp into a float nanosecond timestamp truncated to a fixed precision
-func nsTimestampToFloat(ns int64) float64 {
```

Comment:
now that there is a min value in the DDSketch, this function is no longer needed.